### DISCUSSION TITLE
6.9.5 linkscolor not working

### DISCUSSION THREAD

**Discussion by asdmendes** [joaomlourenco/novathesis#297] [Q&A] [answered]
Hey, professor!
I've tried changing the linkscolor in Config>1_novathesis.tex to any other in the svgnames/xcolor package and the hyperlinks/references just don't change color 😕
I've also tried this same change on version 6.5.3 and it works there!
Thanks so far!

**Comment by joaomlourenco** [comment, marked as answer]
Thanks you for reporting this bug. I created an issue from this discussion and this bug will be fixed in the next release.

**Reply by joaomlourenco** [reply]
Fixed in v6.9.6.
